stage_descriptions/producing-messages-05-sb8.md

```diff
@@ -0,0 +1,43 @@
+In this stage, you'll add support for producing multiple records in a single `Produce` request.
+
+## Producing multiple records
```

Comment:
@ryan-gang at what point do we test "appending" to a log file? That seems like something that'd need extra implementation and something we should explicitly call out and test.